package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@retailmenot/anchor",
-  "version": "1.5.8",
+  "version": "1.6.0",
```

Comment:
Checking NPM, we actually only have released `1.5.7` so I think leaving it at `1.5.8` is probably fine.

Reply:
I bumped up to 1.6.0 because I consider it a "breaking" change. (i.e. Your buttons won't space properly if you were relying on that style)